packages/flutter/lib/src/widgets/selectable_region.dart

```diff
@@ -2367,6 +2372,7 @@ abstract class MultiSelectableSelectionContainerDelegate extends SelectionContai
       final SelectionResult childResult = dispatchSelectionEventToChild(child, event);
       switch (childResult) {
         case SelectionResult.next:
+        case SelectionResult.forward:
```

Comment:
`SelectionResult.forward` is not returned by any of the edge update events at the moment so I choose to handle them here. I could also throw an unimplemented error here instead.